src/IECoreRI/RendererImplementation.cpp

```diff
@@ -1828,44 +1828,72 @@ void IECoreRI::RendererImplementation::emitCurvesPrimitive( const IECore::Curves
 namespace
 {
 
-/// \todo: perhaps this should be made public in MeshAlgo or PrimitiveAlgo
-void convertUVs( PrimitiveVariableMap &variables )
-{
-	std::set<Data *> visited;
-
-	for( auto it = variables.begin(), eIt = variables.end(); it != eIt; ++it )
-	{
-		if( visited.find( it->second.data.get() ) != visited.end() )
+// As of Cortex 10, we take a UDIM centric approach
+// to UVs, which clashes with RenderMan, so we must
+// flip the t values before sending to the renderer.
+// We also represent UVs as V2fVectorData and must
```

Comment:
RenderMan should accept `float[2]` for st I think.

Reply:
What??? Really? So we've been living a lie all these years?

Reply:
Yeah. It's my fault.

Reply:
Addressed in 2b7bdbd6460120fb3077ed8929f5e1fa009460b4